_posts/2021-12-15-Introducing-Data-Prepper-1.2.0-with-Log-Pipelines.md

```diff
@@ -0,0 +1,141 @@
+---
+layout: post
+title:  "Introducing Data Prepper 1.2.0 with Log Pipelines"
+authors:
+- dlv
+date: 2021-12-15 14:00:00 -0500
+categories:
+  - technical-post
+twittercard:
+  description: " You can start using Data Prepper 1.2.0 for log ingestion today. This release enables log collection into OpenSearch from Fluent Bit."
+---
+
+Development teams are using Data Prepper and OpenSearch to better understand their applications through 
+[Trace Analytics](https://opensearch.org/docs/latest/observability-plugins/trace/index/). Now teams can 
+use Data Prepper and OpenSearch for log analysis as well. You can start using Data Prepper 1.2.0 for 
+log ingestion today. This release enables log collection into OpenSearch from 
+[Fluent Bit](https://fluentbit.io/), supports Logstash configuration files, and provides other improvements.
+
+## Log Ingestion
+
+### The Problem with Logs
+
+OpenSearch is a great tool for searching log data and developers are already using it for log analysis. 
+Modern applications are distributed, often containing multiple services running across different servers. 
+Developers want to analyze their log data centrally to more quickly address customer issues and improve 
+application performance, availability, and security.
+
+Getting log data into OpenSearch isn’t easy though. Application logs are generally unstructured data. 
+They are lines of text which humans can read, but are not great for searching and analysis at large scale. 
+Developers need to structure their log data in OpenSearch, enabling then to search for key events. Say, for 
+example, finding slow HTTP requests during a certain time window.
+
+### The Solution
+
+Data Prepper now supports log ingestion into OpenSearch using [FluentBit](https://fluentbit.io/). FluentBit 
+is a popular Apache-licensed log forwarder. It runs alongside an application, reads log files, and forwards them 
+to a destination over HTTP. Data Prepper receives these log lines using the new 
+[http source plugin](https://github.com/opensearch-project/data-prepper/tree/main/data-prepper-plugins/http-source).
+Data Prepper saves each log line as an individual OpenSearch document.
+
+This diagram outlines the basic architecture for log ingestion using FluentBit, Data Prepper, and OpenSearch.
+
+![Architecture](/assets/media/blog-images/2021-12-15-Introducing-Data-Prepper-1.2.0-with-Log-Pipelines/Architecture.png)
```

Comment:
Add `{: .img-fluid}` to the end of the markdown image link. This will make the image expand/contract with the template.

Current state can break out of the column:

![Screen Shot 2021-12-13 at 2 29 47 PM](https://user-images.githubusercontent.com/1152927/145891880-f73be2ac-b523-4c04-a722-1dc37e0d6ebb.png)

Reply:
I added these, thanks!